### PR TITLE
Add Bevy Engine to list of repos

### PR DESCRIPTION
**Pull request by alice-i-cecile**
This a Rust-based open source game engine :)

Issue tracker link: https://github.com/bevyengine/bevy/issues?q=is%3Aopen+is%3Aissue+label%3AE-Good-First-Issue

#### ℹ️ Repository information

**The repository has**:

- [x] At least three issues with the `good first issue` label.
- [x] At least 10 contributors.
- [x] Detailed setup instructions for the project.
- [x] CONTRIBUTING.md
- [x] Actively maintained.

### CODE REVIEW

**Review by siddhant-deepsource — CHANGES_REQUESTED** [review]
Hey @alice-i-cecile, thanks for contributing 🙌 Would be great if you can add the repository name in lexicographic order i.e. after `github.com/beautify-web/js-beautify`. Also, please replace the backticks from the repository name with single quotes. Thanks!

File: data/repositories.toml [hunk 1]
```diff
@@ -75,6 +75,7 @@ repositories = [
   'github.com/bastion-rs/bastion',
   'github.com/bazelbuild/bazel',
   'github.com/bbatsov/projectile',
+  `github.com/bevyengine/bevy`,
```
Comment (alice-i-cecile) [inline data/repositories.toml:78]:
```suggestion
```

File: data/repositories.toml [hunk 2]
```diff
@@ -75,6 +75,7 @@ repositories = [
...
   'github.com/bcherny/json-schema-to-typescript',
   'github.com/beautify-web/js-beautify',
```
Comment (alice-i-cecile) [inline data/repositories.toml:80]:
```suggestion
  'github.com/beautify-web/js-beautify',
  'github.com/bevyengine/bevy',
```

**Comment by siddhant-deepsource** [comment]
Hey @alice-i-cecile, thanks for making the change. As of now we only support fetching and displaying the issues labelled as `good first issue`. Just checked the repository issues and since they are labelled as `E-Good-First-Issue` so currently they would not be displayed on the dashboard :( .Thus, closing this PR for now. Thanks for contributing anyway! 🙌

**Pull request closed without merging**